tests/unit/conftest.py

```diff
@@ -190,6 +190,22 @@ def application_instance(db_session):
     return application_instance
 
 
+@pytest.fixture
+def lti13_application_instance(db_session):
+    lti_registration = factories.LTIRegistration()
+
+    application_instance = factories.ApplicationInstance(
```

Comment:
not reusing `application_instance`, we want two different rows in the DB.

Reply:
Might be worth keeping an eye on this to see if we end up using it in more than one place. Or alternatively we could move it where it's used and ship it outwards as it gets more widely used?

Reply:
As is relatively big for a fixture I think it makes sense to be close to the v11 version but it's also true that it's only used once now.